.drone.yml

```diff
@@ -114,7 +115,7 @@ steps:
       - bundle exec rake payload:send[frontend,$latest_image,http://master.devkube.com:1337]
```

Comment:
Let's trigger this one only on master branch for now
Later we can do $DRONE_BRANCH.devkube.com